src/mui/list/List.spec.js

```diff
@@ -21,6 +21,38 @@ describe('<List />', () => {
         translate: () => {},
     };
 
+    describe('Content Values', () => {
+      it('check the no results text is displayed correctly', () => {
+        const translateSpy = sinon.spy();
+        const wrapper = shallow(
+            <List
+                {...defaultProps}
+                translate={translateSpy}
+                children={[]}
+                total={0}
+            >
+                <div />
+            </List>
+        );
+
+        assert.equal(translateSpy.calledWith('aor.list.no_results'), true);
```

Comment:
I'd prefer that you use the `ShallowWrapper` and `findElement` to test the result. Your current test is testing the implementation.